demoapp/config.xml

```diff
@@ -29,13 +28,12 @@
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
     </platform>
-    <plugin name="cordova-plugin-whitelist" spec="^1.3.3" />
-    <plugin name="cordova-plugin-file" spec="^4.0.0" />
+    <plugin name="cordova-plugin-whitelist" spec="^1.3.4" />
+    <plugin name="cordova-plugin-file" spec="^5.0.0" />
     <plugin name="cordova-plugin-camera" spec="^3.0.0" />
-    <plugin name="cordova-plugin-appcenter-analytics" spec="^0.3.6" />
-    <plugin name="cordova-plugin-appcenter-crashes" spec="^0.3.6" />
-    <plugin name="cordova-plugin-appcenter-push" spec="^0.3.6" />
-    <plugin name="cordova-plugin-appcenter-shared" spec="^0.3.6" />
-    <engine name="android" spec="^7.0.0" />
-    <engine name="ios" spec="5.0.0" />
+    <plugin name="cordova-plugin-appcenter-shared" spec="file:../cordova-plugin-appcenter-shared" />
```

Comment:
Same here.
Please also be careful when updating basic plugins versions (like **file** plugin).
Changes like this should be tested cause they can produce some flaky behavior on cordova.
So I suppose the binary attachments to be tested after this.